src/utils/hash_sent.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import hashlib
+import math
+
+#This code provides a piecewise hashing of documents with variables to set the window_size for each piece, whether to split on words or characters, whether to provide sliding, and how big the sliding_window should be.
+
+#Functions
+
+#hashDocument:
+	#Parameters:
+		#text: text of document to be hashed
+		#window_size: default of 25; number of characters or words to hash at a time
+		#words: default of False; hash by characters = False; hash by words = True
+		#sliding: default of True; slide the window and overlap the hashes or have no overlap
+		#sliding_window: default of 4; ignored if sliding = False; must be less than the size of window_size
+	#Description: Hashes a text document using a window_size to hash piecewise. Provide string of text document and string of piecewise hash for that text document will be returned.
+
+#splitText:
+	#Parameters:
+		#text: text of document to be hashed
+		#window_size: default of 25; number of characters or words to hash at a time
+		#words: default of False; hash by characters = False; hash by words = True
+		#sliding: default of True; slide the window and overlap the hashes or have no overlap
+		#sliding_window: default of 4; ignored if sliding = False; must be less than the size of window_size
+	#Description: Splits the text into the sizes needed for hashing. Returns a list of strings split based on the parameters provided
+
+def hashDocument(text, window_size = 25, words = False, sliding = True, sliding_window = 4):
```

Comment:
if doc length is less than window_size - sliding_window i get blank output. is that the correct behavior?